### ISSUE TITLE
Error: Call to a member function getLogs() on null

### ISSUE THREAD

**Issue by bgeneto** [daycry/cronjob#28]
Hello! After the latest upgrade I'm getting the followin while trying to access the logs (gear icon): 
``` 
Call to a member function getLogs() on null
VENDORPATH/daycry/cronjob/src/Traits/LogTrait.php at line 64
return $this->handler->getLogs($this->name);
```

Also I cannot run my migrations anymore: 

```
php spark migrate --all 
[CodeIgniter\Database\Exceptions\DatabaseException]
Duplicate column name 'context'
at SYSTEMPATH/Database/BaseConnection.php:647
```
Because I'm using CI 4.4.6 settings package (required by shield) and it has a `AddContextColumn` migration class that adds this `context` column already. 

Also, what `public string $papa = 'file';` means in the new config file?

TIA.

**Comment by daycry** [comment]
I Will check this error.

I means that is muy mistake.

I Will try to refactor this Code.

**Comment by daycry** [comment]
Error migration fixed in 2.2.21, integrated with Codeigniter4/settings for more compatibilities with other vendors like SHIELD.
